srcpkgs/snapper/patches/0001-disable-locale-tests.patch

```diff
@@ -0,0 +1,28 @@
+diff --git testsuite/Makefile.am testsuite/Makefile.am
+index d7e30b8..001f834 100644
+--- testsuite/Makefile.am
++++ testsuite/Makefile.am
+@@ -7,8 +7,8 @@ AM_CPPFLAGS = -I$(top_srcdir) $(DBUS_CFLAGS)
+ LDADD = ../snapper/libsnapper.la ../dbus/libdbus.la -lboost_unit_test_framework
+ 
+ check_PROGRAMS = sysconfig-get1.test dirname1.test basename1.test 		\
+-	equal-date.test dbus-escape.test cmp-lt.test humanstring.test 		\
+-	table.test table-formatter.test csv-formatter.test json-formatter.test	\
++	equal-date.test dbus-escape.test 		\
++	csv-formatter.test json-formatter.test	\
+ 	getopts.test
+ 
+ if ENABLE_BTRFS_QUOTA
+@@ -27,12 +27,6 @@ EXTRA_DIST = $(noinst_SCRIPTS) sysconfig-get1.txt sysconfig-set1.txt
+ 
+ equal_date_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la
+ 
+-humanstring_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la
+-
+-table_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la
+-
+-table_formatter_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la
+-
+ csv_formatter_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la
+ 
+ json_formatter_test_LDADD = -lboost_unit_test_framework ../client/utils/libutils.la $(JSONC_LIBS)
```

Comment:
Any idea why these fail?

Reply:
Here is the error (for all 4 of them):
`std::runtime_error: locale::facet::_S_create_c_locale name not valid`

Honestly I have no idea

Reply:
Ah, that's a libstdc++ on musl thingy. Thanks. I will try to get them worked out upstream. Could you include the error string above the patch?

Reply:
Alternatively, try doing `do_check() { LANG=C make check }`

Reply:
Yes will do after I tinker with json-c

Reply:
Hm, shame. It had been enough to fix `qpdf`, whose test suite had similar issues.

Reply:
I believe this issue doesn't happen on musl alone. CI failed for x86-64 and i386 as well.
I can't test locally on those archs sadly to confirm.

Reply:
You can make a masterdir for `x86_64` with `./xbps-src binary-bootstrap x86_64`.

Reply:
Thanks. Those tests fail locally in x86_64 masterdir for me.